.babelrc

```diff
@@ -1,4 +1,12 @@
 {
   "presets": ["next/babel"],
-  "plugins": [["styled-components", { "ssr": true }]]
+  "plugins": [
+    ["styled-components", { "ssr": true }],
+    [
+      "@babel/plugin-transform-react-jsx",
```

Comment:
Not needed, see https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#nextjs.

Reply:
Thanks!